addons/hr_skills/static/src/scss/report_employee_cv.scss

```diff
@@ -28,37 +17,31 @@ div.o_employee_cv {
     }
 
     .o_sidebar {
-        color: #fff;
+        color: $white;
```

Comment:
Still not working on the runbot.
We might want to keep the hexa code.

[See here](https://zupimages.net/up/23/13/edqr.jpg)

Reply:
Yes, I think you should squash commits related to this.